app/components/modals/tax-info-modal.js

```diff
@@ -21,16 +21,18 @@ export default ModalBase.extend(FormMixin, {
       delay  : false,
       on     : 'blur',
       fields : {
-        tax_name: {
-          rules: [
+        taxName: {
+          identifier : 'tax_name',
+          rules      : [
             {
               type   : 'empty',
               prompt : this.i18n.t('Please give a name')
             }
           ]
         },
-        tax_rate: {
-          rules: [
+        taxRate: {
+          identifier : 'tax_rate',
+          rules      : [
             {
               type   : 'empty',
               prompt : this.i18n.t('Please tell us your tax rate (in %)')
```

Comment:
@geekyd No. That is necessary. We don't need exponential percentages 😉 
![image](https://user-images.githubusercontent.com/17252805/27392485-6c7381cc-56c4-11e7-815d-895b21fa9bbb.png)
https://stackoverflow.com/questions/31706611/why-does-the-html-input-with-type-number-allow-the-letter-e-to-be-entered-in/31706796